src/hb-ot-kern-table.hh

```diff
@@ -205,7 +205,7 @@ struct KernSubTableWrapper
   {
     TRACE_SANITIZE (this);
     return_trace (c->check_struct (thiz()) &&
-		  thiz()->length >= thiz()->min_size &&
+		  thiz()->length >= T::min_size &&
```

Comment:
Needed as the original issue linking issue,

```
  GEN      libharfbuzz.la
Undefined symbols for architecture x86_64:
  "OT::KernOT::SubTableWrapper::min_size", referenced from:
      OT::KernSubTableWrapper<OT::KernOT::SubTableWrapper>::sanitize(OT::hb_sanitize_context_t*) const  in libharfbuzz_la-hb-ot-font.o
  "OT::KernAAT::SubTableWrapper::min_size", referenced from:
      OT::KernSubTableWrapper<OT::KernAAT::SubTableWrapper>::sanitize(OT::hb_sanitize_context_t*) const  in libharfbuzz_la-hb-ot-font.o
ld: symbol(s) not found for architecture x86_64
collect2: ld returned 1 exit status
```

Reply:
Ok, good. So you didn't have to duplicate sanitize after all.